Options.py

```diff
@@ -394,7 +393,7 @@ def __ne__(self, other):
     __hash__ = Option.__hash__  # see https://docs.python.org/3/reference/datamodel.html#object.__hash__
 
 
-class TextChoice(FreeText, Choice):
+class TextChoice(Choice):
```

Comment:
you sure? :D

Reply:
turns out i wasn't actually using anything from freetext as all of the special handling in it i'm overriding anyways